docs/reference/search/request/sort.asciidoc

```diff
@@ -82,6 +82,10 @@ to. The `mode` option can have the following values:
 `median`:: Use the median of all values as sort value.  Only applicable
            for number based array fields.
 
+The default sort mode in the ascending sort order is `min` - the lowest value
```

Comment:
Super small comment: I think this should be "`min` -- the lowest value" so it shows up as a dash instead of a hyphen.